cypress/e2e/page-list.spec.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @copyright Copyright (c) 2023 Max <max@nextcloud.com>
+ *
+ * @author Max <max@nextcloud.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ *  Tests for the page list.
+ */
+
+describe('Page list', function() {
+	before(function() {
+		cy.login('bob', { route: '/apps/collectives' })
+		cy.deleteAndSeedCollective('Our Garden')
+		cy.seedPage('Day 1', '', 'Readme.md')
+		// Wait 1 second to make sure that page order by time is right
+		cy.wait(1000) // eslint-disable-line cypress/no-unnecessary-waiting
+		cy.seedPage('Day 2', '', 'Readme.md')
+		cy.seedPage('Page Title', '', 'Readme.md')
+		cy.seedPage('Subpage Title', '', 'Day 1.md')
+		cy.seedPage('#% special chars', '', 'Readme.md')
+		cy.seedPageContent('Our Garden/Day 2.md', 'A test string with Day 2 in the middle and a [link to Day 1](/index.php/apps/collectives/Our%20Garden/Day%201).')
```

Comment:
This line can probably be removed here, given that we need the page content only for the backlinks test in `page-details.spec.js` and the fulltext search test in `pages.spec.js`.

Reply:
I took the liberty to push a commit that removes some superfluous page creations.